src/witness/yamlWitnessType.ml

```diff
@@ -0,0 +1,330 @@
+module Producer =
+struct
+  type t = {
+    name: string;
+    version: string;
+    (* TODO: configuration *)
+    command_line: string;
+    (* TODO: description *)
```

Comment:
Are these todos that need to be taken care of before merging this PR?

Reply:
No, they were also present (just in a different place) before this PR. Those are just optional metadata fields that we currently don't need/use.